src/rpdk/core/test.py

```diff
@@ -341,6 +342,7 @@ def get_contract_plugin_client(args, project, overrides, inputs):
         project.type_name,
         args.log_group_name,
         args.log_role_arn,
+        headers=(args.source_account, args.source_arn),
```

Comment:
I believe headers will always not be None and have a length of 2 in this case (None, None) so it would always go into the inject_headers in boto_helpers.py

Reply:
Good catch.  (None, None) will still be a length of 2.